blog/v4_announcement.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>Maybe add <code>aehmc</code>, <code>aemcmc</code> ?</p><p> </p><p>Also we may want to add here, that we now have a pymc-experimental for  newly-implemented statistical methods and code not yet useful in "production"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc.io/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
+1 on mentioning pymc-experimental! We already had a couple questions on discourse about BART which is now there for example